OnlineStore/store/src/main/java/Store.java

```diff
@@ -0,0 +1,96 @@
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toCollection;
+
+public class Store {
+
+    ArrayList<Product> products;
```

Comment:
when u declare collection, it is better to use top level inteface or class, in this case - List